stacks/cloud-manager/deploy.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/sh
+
+set -e
+
+################################################################################
+# repo
+################################################################################
+helm repo add ei https://cdn.einnovator.org/charts
```

Comment:
When I attempt to run this, I get the following:
```
Error: looks like "https://cdn.einnovator.org/charts" is not a valid chart repository or cannot be reached: Get "https://cdn.einnovator.org/charts/index.yaml": dial tcp: lookup cdn.einnovator.org: no such host
```

Reply:
Hi, @paulanunda !
I just run the command both from my windows and from my mac and it worked.
helm repo add ei https://cdn.einnovator.org/charts
outputs:
"ei" has been added to your repositories
Can you please try again..
The file your error message is complaining is there:
https://cdn.einnovator.org/charts/index.yaml
Copy this URL on the browser and do a curl from the machine you are running to double check.
Please let me know if the issue still persists.
Thanks,
Jorge.

Reply:
Also try:
ping cdn.einnovator.org
to see if it is a IP connectivity problem.
Thanks!